src/main/java/com/geronimo/model/Message.java

```diff
@@ -0,0 +1,46 @@
+package com.geronimo.model;
+
+import lombok.Data;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedDate;
+import org.springframework.data.annotation.Version;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.*;
+
+@Data
+@Entity
+@Table(name = "messages")
+@EntityListeners(AuditingEntityListener.class)
+public class Message {
```

Comment:
Why did you remove extension from AuditedEntity? Doesn't it works just to put this listener there and have these dateCreated, lastUpdated fields annotated there?

Reply:
my bad